tests/codegen/repr/transparent.rs

```diff
@@ -43,7 +43,6 @@ pub extern "C" fn test_WithZst(_: WithZst) -> WithZst { loop {} }
 #[repr(transparent)]
 pub struct WithZeroSizedArray(*const f32, [i8; 0]);
 
-// Apparently we use i32* when newtype-unwrapping f32 pointers. Whatever.
```

Comment:
This comment seems to be from a time when LLVM still had pointee types, so I think it can be removed.